src/main/scala/ai/privado/utility/ImportUtility.scala

```diff
@@ -0,0 +1,111 @@
+package ai.privado.utility
+
+import scala.util.control.Breaks._
+import java.nio.charset.MalformedInputException
+import scala.io.Source
+import io.shiftleft.codepropertygraph.generated.Languages
+
+object ImportUtility {
+
+  private var allImports: Set[String] = Set();
+
+  // get language specific import matching regex
+  private def getMatchImportRegex(language: String): String = {
+    val result = language match {
+      case Languages.JAVA => "^\\s*(import)\\s+.*$"
+      case Languages.PYTHON => "^\\s*(import)\\s+.*$"
+      case Languages.JAVASCRIPT => "^\\s*(import|require)\\s+.*$"
+      case default => "(import)\\s+" // Default is java
+    }
+    return result;
+  }
+
+  private def getFileExtension(language: String): String = {
+    val result = language match {
+      case Languages.JAVA => ".java"
+      case Languages.PYTHON => ".py"
+      case Languages.JAVASCRIPT => ".js"
+      case default => ".java" // Default is java
+    }
+
+    return result;
+  }
+
+  // get regex to match only import words
+  private def getLiteralWordRegex(language: String): String = {
+    val result = language match {
+      case Languages.JAVA => "(import)\\s+"
+      case Languages.PYTHON => "(import)\\s+"
+      case Languages.JAVASCRIPT => "(import|require)\\s+"
+      case default => "(import)\\s+" // Default is java
+    }
+    return result;
+  }
+
+  private def returnImportStatements(filePath: String, language: String): Set[String] = {
+    val source = Source.fromFile(filePath)
```

Comment:
@karan-batavia Include this step inside inside the below Try block the statement `source.close()` on Line number 83 inside `finally`